### PR TITLE
Update FrontendOptions.java : add some log

### PR DESCRIPTION
**Pull request by mklzl**
print some logs to remind users to properly configure Priority_networks #349

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/service/FrontendOptions.java [hunk 1]
```diff
@@ -77,6 +77,10 @@ public static void init() throws UnknownHostException {
                         localAddr = addr;
                         break;
                     }
+                    else {
```
Comment (imay) [inline fe/fe-core/src/main/java/com/starrocks/service/FrontendOptions.java:80]: in the same line
```suggestion
                    } else {
```

Comment (mklzl) [reply fe/fe-core/src/main/java/com/starrocks/service/FrontendOptions.java]: I fixed the mistake

**Review by imay — DISMISSED** [review]

**Review by Astralidea — DISMISSED** [review]

**Comment by gengjun-git** [comment]
Hello mklzl, welcome to your contribution. Can you improve the commit: if all ips not match the priority_networks then print the warning log, or just exit. Machine may has multi ip address, and the unmatched ip may confuse the user.

**Review by gengjun-git — COMMENTED** [review]
Hello mklzl, welcome to your contribution. Can you improve the commit: if all ips not match the priority_networks then print the warning log, or just exit. Machine may has multi ip address, and the unmatched ip may confuse the user.

**Comment by mklzl** [comment]
> improve

Thanks for your advice, I will improve this commit

**Review by gengjun-git — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/java/com/starrocks/service/FrontendOptions.java [hunk 2]
```diff
@@ -75,15 +76,19 @@ public static void init() throws UnknownHostException {
...
+                    } 
                 } else {
                     localAddr = addr;
                     break;
                 }
             }
         }
-
+        //if all ips not match the priority_networks then print the warning log
+        if (!checkIpFlag) {
```
Comment (imay) [inline fe/fe-core/src/main/java/com/starrocks/service/FrontendOptions.java:89]: When priority_network is not set, checkIpFlag is false, this branch will execute. That is not the actual intent.
So I think priorityCidrs.isEmpty() should also be checked here.

About the flag name, I prefer hasMatchedIp.

Comment (mklzl) [reply fe/fe-core/src/main/java/com/starrocks/service/FrontendOptions.java:89]: Okay, let me modify my code

**Review by gengjun-git — APPROVED** [review]

**Review by imay — APPROVED** [review]

**Pull request merged**